domain/flows/order_gift.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.1"
+
+slots:
+  order_gift_confirmation:
+    type: bool
+    mappings:
+    - type: from_llm
+  gift_recipient:
+    type: text
+    mappings:
+    - type: from_llm
+    validation:
+      rejections:
+        - if: slots.gift_recipient == "me"
+          utter: utter_gift_recipient_invalid
+  gift_name:
+    type: text
+    mappings:
+      - type: from_llm
+  gift_price:
+    type: float
+    mappings:
+      - type: from_llm
+    validation:
+      rejections:
+        - if: slots.gift_price < 100
+          utter: utter_gift_price_invalid
+
+responses:
+  utter_ask_order_gift_confirmation:
+    - text: Do you want to buy a {gift_name} worth {gift_price} for {gift_recipient}?
+      buttons:
+        - payload: yes
+          title: "Yes"
+        - payload: no
```

Comment:
Could we use `/SetSlots` syntax - makes the e2e test less flaky on the long run.